### DISCUSSION TITLE
Can't start a new build: stuck on "branches are loading"

### DISCUSSION THREAD

**Discussion by andyNroses** [codemagic-ci-cd/codemagic-docs#1268] [Q&A] [answered]
Hi!

Since yesterday, I cannot start any new build as the modal is stuck with "branches are loading". After inspecting the branches API call, it seems to return an odd response (an id string instead of what I would expect to be an array or branches), but perhaps the problem is entirely elsewhere.

![Screenshot_20221225_125039_Chrome](https://user-images.githubusercontent.com/9218506/209466933-cb749280-6db1-4afe-90ba-158ba859477d.jpg)

Could anyone help? 

Thanks.

**Comment by andyNroses** [comment]
Update: this seems to be related to same issue as #1267 as I'm getting the same error when re-logging in with GitLab

**Comment by K-Codemagic** [comment, marked as answer]
Hi, it should work now, however, make sure you log out / log in to your account to refresh OAuth token. Without it, builds will fail to clone the repository. Sorry for the inconvenience caused.

**Reply by andyNroses** [reply]
It works! Thanks a lot.
